app/views/early_career_payments/claims/induction_completed.html.erb

```diff
@@ -0,0 +1,75 @@
+<% content_for(:page_title, page_title(t("early_career_payments.questions.induction_completed.heading"), policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
+<% path_for_form = current_claim.persisted? ? claim_path(current_policy_routing_name) : claims_path(current_policy_routing_name) %>
+<% shared_view_css_size = current_claim.policy == EarlyCareerPayments ? "l" : "xl" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { "eligibility.induction_completed": "claim_eligibility_attributes_induction_completed_true" }) if current_claim.errors.any? %>
```

Comment:
Is this a GitHub view or the line is actually breaking into two lines?

Reply:
@martin-dfe this `render` statement is on one line; pretty standard, I copied it from other similar views.